pkg/occlient/occlient.go

```diff
@@ -262,6 +261,15 @@ func isServerUp(server string) bool {
 		return false
 	}
 
+	ocRequestTimeout := config.DefaultTimeout * time.Second
```

Comment:
we can skip this line as `cfg.GetTimeout()` will return the Default value in the else statement

Reply:
No, this is the case `if the config file is not able to read`, it wont fail instead proceed with the default value

Reply:
`unable to read config file` will be logged on odo cmd

Reply:
OK but please add a comment describing the use of this line so that in future it does not get removed my accident.